008_team_6_agent_design/008_team_6_agent_design.tex

```diff
@@ -1 +1,1014 @@
+    % \includegraphics[width=0.3\linewidth]{C/C1_utility.pdf}\\[0pt]
+    % (a) C1: social motives & (b) C1: deaths over time & (c) C1: utility over time \\[8pt]
+    \includegraphics[width=0.3\linewidth]{008_team_6_agent_design/C/C2_SM.pdf} &   \includegraphics[width=0.3\linewidth]{008_team_6_agent_design/C/C2_deaths.pdf} &
+    \includegraphics[width=0.3\linewidth]{008_team_6_agent_design/C/C2_utility.pdf}\\[0pt]
+    (a) C1: social motives & (b) C1: deaths over time & (c) C1: utility over time \\[8pt]
+    % \includegraphics[width=0.3\linewidth]{C/C3_SM.pdf} &   \includegraphics[width=0.3\linewidth]{C/C3_deaths.pdf} &
+    % \includegraphics[width=0.3\linewidth]{C/C3_utility.pdf}\\[0pt]
+    % (g) C3: social motives & (h) C3: deaths over time & (i) C3: utility over time \\[8pt]
+    \includegraphics[width=0.3\linewidth]{008_team_6_agent_design/C/C4_SM.pdf} &   \includegraphics[width=0.3\linewidth]{008_team_6_agent_design/C/C4_deaths.pdf} &
+    \includegraphics[width=0.3\linewidth]{008_team_6_agent_design/C/C4_utility.pdf}\\[0pt]
+    (d) C2: social motives & (e) C2: deaths over time & (f) C2: utility over time \\[8pt]
+    \includegraphics[width=0.3\linewidth]{008_team_6_agent_design/C/C5_SM.pdf} &   \includegraphics[width=0.3\linewidth]{008_team_6_agent_design/C/C5_deaths.pdf} &
+    \includegraphics[width=0.3\linewidth]{008_team_6_agent_design/C/C5_utility.pdf}\\[0pt]
+    (g) C3: social motives & (h) C3: deaths over time & (i) C3: utility over time \\[8pt]
+    \end{tabular}
+    \caption{Simulation results for case C. C1 and C3 include communication, but C2 does not. C3 includes a more restrictive treaty.}
+    \label{fig:res_C}%
+\end{figure}
+
+% C2 was renamed C1, C4 renamed C2. We dropped the initial  
+
+
+% C4 = less stable than C2
+% C4 has death increase after day 30 = reshuffle
+% Narcissist can survive with no comms, acting individually - get killed with partial comms in C2 (unlike D with forced comms, where they must act collectivist)
+
+
+The overarching comment to draw from this set of results is the impact of specific treaties on the global utility. By comparing the results C1 and C2, the system using the `classical' collectivist treaty C1 (see \Cref{proposing_treaties}) yields a lower performance than the system without treaties C2: the number of deaths is lower in C2, and its global utility is instead higher.
+
+To explain these rather counter-intuitive results, we offer the following hypothesis. As the agents' health falls, their social motives tend to change toward narcissist. In the case of collectivists having signed a treaty, they will turn selfish. Instead of following the natural decision of this social motive (eat to stay above the healthy level), they have to follow the treaty. The moment their health falls below the \textit{weakLevel}, the treaty they signed no longer applies and they will follow their natural food intake rule defined in \Cref{food_consumption}:
```

Comment:
```suggestion
To explain these rather counter-intuitive results, we offer the following hypothesis. As the agents' health falls, their social motives tend to change toward narcissist. In the case of collectivists having signed a treaty, they will turn selfish. Instead of following the natural decision of this social motive (eat to stay above the healthy level), they have to follow the treaty. The moment their health falls below the \texttt{weakLevel}, the treaty they signed no longer applies and they will follow their natural food intake rule defined in \Cref{food_consumption}:
```